Formula/php@8.0.rb

```diff
@@ -0,0 +1,471 @@
+class PhpAT80 < Formula
+  desc "General-purpose scripting language"
+  homepage "https://www.php.net/"
+  # Should only be updated if the new version is announced on the homepage, https://www.php.net/
+  url "https://www.php.net/distributions/php-8.0.13.tar.xz"
+  mirror "https://fossies.org/linux/www/php-8.0.13.tar.xz"
+  sha256 "cd976805ec2e9198417651027dfe16854ba2c2c388151ab9d4d268513d52ed52"
+  license "PHP-3.01"
+  revision 1
+
+  livecheck do
+    url "https://www.php.net/downloads"
+    regex(/href=.*?php[._-]v?(\d+(?:\.\d+)+)\.t/i)
+  end
+
+  bottle do
+    sha256 arm64_monterey: "5e8e562c74584798172ad38b133ebb697d2aad05ef864943b03775edb00677ce"
+    sha256 arm64_big_sur:  "65f85437a8c83d56ecc117d80608ea888031defae926fd39422c2c17a41b6527"
+    sha256 monterey:       "77bf294ebd73b374e914a4224a050698b1a2534a5bc39b01b600a28380b83720"
+    sha256 big_sur:        "b85115ff05fcae0b688d3fd941c99991511748d720ff36e5cc3147fc9d849afa"
+    sha256 catalina:       "47c1c304eb20b686f097841095b8c14d75d51dbd385532971a234a14fe2a0f6a"
+    sha256 x86_64_linux:   "dcbe5be8ab4acff8dd28ccab2eb59dfb642c49ec6e86645485bb447d75f04aba"
+  end
+
+  keg_only :versioned_formula
+
+  depends_on "httpd" => [:build, :test]
+  depends_on "pkg-config" => :build
+  depends_on "apr"
+  depends_on "apr-util"
+  depends_on "argon2"
+  depends_on "aspell"
+  depends_on "autoconf"
+  depends_on "curl"
+  depends_on "freetds"
+  depends_on "gd"
+  depends_on "gettext"
+  depends_on "gmp"
+  depends_on "icu4c"
+  depends_on "krb5"
+  depends_on "libffi"
```

Comment:
We should make this `uses_from_macos "libffi", since: :catalina` too.

Reply:
Done